rosdep/base.yaml

```diff
@@ -595,6 +596,7 @@ eigen:
     stretch: [libeigen3-dev]
     wheezy: [libeigen3-dev]
   fedora: [eigen3-devel]
+  freebsd: [eigen]
```

Comment:
I don't see plain eigen listed: https://www.freshports.org/search.php?stype=name&method=match&query=eigen&num=10&orderby=category&orderbyupdown=asc&search=Search&format=html&branch=head I see eigen2 and eigen3

Reply:
Actually the field you want to look at from freshports is **PKGNAME**. That's the name the package program uses. It doesn't necessarily line up with the port name. I'm not sure why this can diverge, but I'm sure there are historical reasons (FreeBSD ports were around long before the new pkg system, and they serve as the backbone for most of pkg) 

In the case above, you'll see that eigen3's package name is, indeed, eigen. So, that should be OK.

I have removed the ros-urdfdom changes for the moment as I would need to push those ports forward. I've got the basics done. I just need to ask to get them added.